Mathlib/FieldTheory/PurelyInseparable.lean

```diff
@@ -867,6 +874,7 @@ namespace Field
 as a natural number. This means that the cardinality of `Field.Emb F E` and the degree of
 `(separableClosure F E) / F` are both finite or infinite, and when they are finite, they
 coincide. -/
+@[stacks 09HJ "The definition of both side in 09HJ is exaclty the same as in this lemma."]
```

Comment:
```suggestion
@[stacks 09HJ "The definition of both side in 09HJ is exactly the same as in this lemma."]
```
Or maybe "`sepDegree` is defined as the right hand side of 09HJ"

Reply:
Thanks. Fixed.